vtex-apps/quickorder
# QTA-138-Added prop alwaysShowAddToCart  show add to cart button quick order

…invalid SKU

#### What does this PR do? \*
An alwaysShowAddToCart property was added when validating whether or not to show the add to cart button. By declaring quickorder-upload in store-theme as true, the button will be shown even if some Sku is invalid. By clicking on the add to cart button, the valid ones will be added and the invalid ones will be ignored. If the property does not exist, validation continues as it was: The button will only be shown if all Sku's are valid.

#### How to test it? \*

Access the workspace  https://qta38--lojaobramax.myvtex.com/quickorder  and add a worksheet with SKUS, valid and invalid.
Click on the add to cart button, which will now appear. Valid items must be added to the cart.

SKU's list
[prods_with_invalids.xlsx](https://github.com/vtex-apps/quickorder/files/12186702/prods_with_invalids.xlsx)





#### Related to / Depends on \*



### Before
[screencast-bpconcjcammlapcogcnnelfmaeghhagj-2023.07.27-15_32_49.webm](https://github.com/vtex-apps/quickorder/assets/116591771/126269a5-7ee6-4750-872d-9ad177ec72a2)

### After
[screencast-bpconcjcammlapcogcnnelfmaeghhagj-2023.07.27-15_30_48.webm](https://github.com/vtex-apps/quickorder/assets/116591771/09e150cb-e0ab-4d5e-b864-35e5bd33912c)
